packages/server-core/src/createApp.ts

```diff
@@ -184,6 +185,12 @@ export const createFeathersExpressApp = (
     return res.status(204).send()
   })
 
+  app.post('/api/credentials/request', IssueRequest.post)
```

Comment:
is there a reason we are not using feathers for this?

Reply:
Main reason is - I don't know how to do it using feathers. Happy to refactor in the next PR tho.